_includes/home/_content-menu.html

```diff
@@ -0,0 +1,15 @@
+<div class="container">
+    <div class="col-md">
+        <section class="content-menu">
+            <p class="menu-title soft-left font-family-serif text-uppercase">Explore All of Our Content</p>
+                <ul class="list-unstyled hard-left soft-quarter-top"> 
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/series">Messages</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/series/">Series</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="https://www.crdsmusic.com/">Music</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/podcasts/">Podcasts</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/articles/">Articles</a></li>
+                    <li class="font-size-large push-half-bottom dashed"><a class="text-orange menu-link" href="/media/videos/">Videos</a></li>
+                </ul>
```

Comment:
Please fix your indentions– you should be using soft-tabs w/ spaces (two spaces per tab).